Barotrauma/BarotraumaClient/ClientSource/Screens/SubEditorScreen.cs

```diff
@@ -36,6 +39,131 @@ public enum Mode
             Default,
             Wiring
         }
+        
+        private GUITickBox RotateToolToggle, ScaleToolToggle;
+        
+        private static Vector2 SelectionCenter
+        {
+            get
+            {
+                List<MapEntity> nonWireEntities = MapEntity.FilteredSelectedList.Where(entity => (entity as Item)?.GetComponent<Wire>() == null).ToList();
```

Comment:
To me it seems unnecessary to instantiate a new list here, could just use the `IEnumerable` returned by `Where`. It seems like this can be called every frame, which means instantiating a new list every time would lead to a considerable performance hit.